<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranscoderOptionsVorbis</class>
 <widget class="QWidget" name="TranscoderOptionsVorbis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string comment="Sound quality">Quality</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QSlider" name="quality_slider">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="quality_spinbox">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="managed">
     <property name="text">
      <string>Use bitrate management engine</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>32</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Target bitrate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="bitrate_slider">
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>128</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="bitrate_spinbox">
          <property name="suffix">
           <string> kbps</string>
          </property>
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>128</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Minimum bitrate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="min_bitrate_slider">
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="min_bitrate_spinbox">
          <property name="specialValueText">
           <string>disabled</string>
          </property>
          <property name="suffix">
           <string> kbps</string>
          </property>
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Maximum bitrate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSlider" name="max_bitrate_slider">
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="max_bitrate_spinbox">
          <property name="specialValueText">
           <string>disabled</string>
          </property>
          <property name="suffix">
           <string> kbps</string>
          </property>
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>quality_slider</tabstop>
  <tabstop>quality_spinbox</tabstop>
  <tabstop>managed</tabstop>
  <tabstop>bitrate_slider</tabstop>
  <tabstop>bitrate_spinbox</tabstop>
  <tabstop>min_bitrate_slider</tabstop>
  <tabstop>min_bitrate_spinbox</tabstop>
  <tabstop>max_bitrate_slider</tabstop>
  <tabstop>max_bitrate_spinbox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>quality_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>quality_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quality_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>quality_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitrate_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bitrate_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitrate_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bitrate_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min_bitrate_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>min_bitrate_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min_bitrate_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>min_bitrate_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max_bitrate_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>max_bitrate_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max_bitrate_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>max_bitrate_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>managed</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
